src/egon/data/metadata.py

```diff
@@ -465,6 +475,16 @@ def sources():
                 "© Europa-Universität Flensburg, Halmstad University and Aalborg University"
             )],
         },
+        "pipeline_classification": {
+                    "title": "Technical pipeline characteristics for high pressure pipelines",
+                    "description": "Parameters for the classification of gas pipelines, "
+                    "the whole documentation could is available at: "
```

Comment:
Is that "could" a quote or is that a typo that should have been something else?

Reply:
Can anyone answer this question?

Reply:
I vote for typo and not necessary.